Integrations/McAfee_Advanced_Threat_Defense/CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+-
+
+## [19.9.0]
```

Comment:
We shouldn't be adding version tags to changelogs, it's messing up the clear release notes script, we handle these cases automatically

Reply:
+1 from bark